### ISSUE TITLE
Uncaught ReferenceError: MutationEvent is not defined

### ISSUE THREAD

**Issue by xigoku** [FlorianRappl/Piral.Blazor#151]
# Bug Report

For more information, see the `CONTRIBUTING` guide.

## Prerequisites

- [x] Can you reproduce the problem in a [MWE](https://en.wikipedia.org/wiki/Minimal_working_example)?
- [x] Are you running the latest version?
- [x] Did you perform a search in the issues?

## Environment Details and Version

[Package Version, Blazor Version, OS, Browser,...]
- Piral.Blazor 8.0.17 (latest from 2024-12-06)
- Edge Version 131.0.2903.70 (Official build) (arm64)
- .net SDK 8.0.30
- npm 10.9.0
- macOS Sequoia 15.1.1

## Description

Every mouse movement on a blazor component produces a exception. `Uncaught ReferenceError: MutationEvent is not defined`.

It's caused by removement of `MutationEvent' type in Chrome. [Link](https://developer.chrome.com/blog/mutation-events-deprecation).

## Steps to Reproduce

1. run [example](https://github.com/smapiot/Piral.Blazor/tree/blazor-8.0/example)
2. open 'localhost:1234' in actual and chromium base browser. Edge in my case.
3. show console
4. hover over counter and move mouse
5. watch exception count rising

<img width="737" alt="Screenshot 2024-12-06 at 11 04 21" src="https://github.com/user-attachments/assets/f786110d-6640-451f-b9d5-2dde50a9d187">

## Expected behavior

No 'Uncaught ReferenceError: MutationEvent is not defined' error in console.

## Actual behavior

A lot of 'Uncaught ReferenceError: MutationEvent is not defined' error in console.

## Possible Origin / Solution

Solution described in [article](https://developer.chrome.com/blog/mutation-events-deprecation).

```js
// Old mutation event usage:  
target.addEventListener('DOMNodeInserted',event => doSomething(event.target));

// Replacement mutation observer code:  
const observer = new MutationObserver(mutationList =>  
  mutationList.filter(m => m.type === 'childList').forEach(m => {  
    m.addedNodes.forEach(doSomething);  
  }));  
observer.observe(target,{childList: true, subtree: true});  
```

**Comment by FlorianRappl** [comment]
This has been addressed with [`piral-blazor` 1.7.2](https://github.com/smapiot/piral/issues/724). What version of `piral-blazor` have you used?

As this is not in this / the NuGet package I'd close the issue. Let me know here / or even better in the referenced issue if the problem persists after using the fixed version.

**Comment by xigoku** [comment]
I used version "piral-blazor": "1.5.3".
The Piral example is set to version "1.5.5".

Thank you for your quick response!
